src/components/ControlPanel/ControlPanel.tsx

```diff
@@ -0,0 +1,36 @@
+import type { FC } from 'react';
+
+import IconSearch from '@icons/search.svg?react';
+
+import styles from './ControlPanel.module.css';
+
+const ControlPanel: FC = () => (
+    <div className={styles.controlPanel}>
+        <div className={styles.searchBar}>
+            <input className={styles.searchInput} type="search" placeholder="Search..." />
+            <button className={styles.searchBtn}>
+                <IconSearch className={styles.searchIcon} />
+            </button>
+        </div>
+
+        <div className={styles.filterBar}>
+            <button className={styles.filterBtn}>Electronics</button>
+            <button className={styles.filterBtn}>Shoes</button>
+            <button className={styles.filterBtn}>Clothes</button>
+        </div>
+
+        <div className={styles.sortBar}>
+            <label className={styles.sortLabel} htmlFor="sort">
+                Sort by:
+            </label>
+            <select className={styles.sortSelect} name="sort" id="sort">
```

Comment:
Please add an arrow 
<img width="467" alt="image" src="https://github.com/snischuk/fe-react-2024/assets/98031768/91e18c83-6d5b-44d9-b2f2-1426f4655ca6">

Reply:
Magic )))
<img width="674" alt="image" src="https://github.com/snischuk/fe-react-2024/assets/98031768/f8918b52-7dbb-4395-826b-d40154f5d82b">

Reply:
please fix in the next pr